### PR TITLE
Review and add new key to Brazilian Portuguese at Forms and Tables.

### PR DESCRIPTION
**Pull request by felipe-balloni**

### CODE REVIEW

File: packages/forms/resources/lang/pt_BR/components.php [hunk 1]
```diff
@@ -2,10 +2,107 @@
...
+
+        ],
+
+    ],
+
+    'markdown_editor' => [
+
+        'toolbar_buttons' => [
+            'attach_files' => 'Anexar arquivo',
+            'bold' => 'negrito',
```
Comment (saade) [inline packages/forms/resources/lang/pt_BR/components.php:65]: Use capital 'N' `Negrito`

File: packages/forms/resources/lang/pt_BR/components.php [hunk 2]
```diff
@@ -2,10 +2,107 @@
...
+        ],
+
+    ],
+
+    'key_value' => [
+
+        'buttons' => [
+
+            'add' => [
+                'label' => 'Adcionar linha',
```
Comment (saade) [inline packages/forms/resources/lang/pt_BR/components.php:38]: Missing 'i' `Adicionar`

File: packages/forms/resources/lang/pt_BR/components.php [hunk 3]
```diff
@@ -2,10 +2,107 @@
...
+
+    'markdown_editor' => [
+
+        'toolbar_buttons' => [
+            'attach_files' => 'Anexar arquivo',
+            'bold' => 'negrito',
+            'bullet_list' => 'Lista',
+            'code_block' => 'Bloco de código',
+            'edit' => 'Editar',
+            'italic' => 'Italico',
```
Comment (saade) [inline packages/forms/resources/lang/pt_BR/components.php:69]: Use accented 'i' `Itálico`

File: packages/forms/resources/lang/pt_BR/components.php [hunk 4]
```diff
@@ -2,10 +2,107 @@
...
+            'link' => 'Link',
+            'ordered_list' => 'Lista numerada',
```
Comment (saade) [inline packages/forms/resources/lang/pt_BR/components.php:71]: This should be `Lista ordenada`

File: packages/forms/resources/lang/pt_BR/components.php [hunk 5]
```diff
@@ -2,10 +2,107 @@
...
+            'preview' => 'Visualisar',
```
Comment (saade) [inline packages/forms/resources/lang/pt_BR/components.php:72]: This should be `Visualizar`

File: packages/forms/resources/lang/pt_BR/components.php [hunk 6]
```diff
@@ -2,10 +2,107 @@
...
+            'strike' => 'Tachado',
+        ],
+
+    ],
+
     'multi_select' => [
         'no_search_results_message' => 'Nenhuma opção corresponde à sua pesquisa.',
         'placeholder' => 'Selecione uma opção',
-        'search_prompt' => 'Sem opções disponívels.',
+        'search_prompt' => 'Digite para iniciar sua pesquisa...',
```
Comment (saade) [inline packages/forms/resources/lang/pt_BR/components.php:81]: `Digite para pesquisar` is a better way of saying that

**Review by saade — CHANGES_REQUESTED** [review]

**Comment by danharrin** [comment]
Marked this as a draft until @saade's comments are resolved.

**Comment by danharrin** [comment]
Thank you!

**Pull request merged**